src/com/goide/runconfig/testing/frameworks/gotest/GotestRunConfigurationProducer.java

```diff
@@ -16,19 +16,10 @@
 
 package com.goide.runconfig.testing.frameworks.gotest;
 
-import com.goide.psi.GoFunctionOrMethodDeclaration;
-import com.goide.psi.GoMethodDeclaration;
 import com.goide.runconfig.testing.GoTestRunConfigurationProducerBase;
-import org.jetbrains.annotations.Nullable;
-
 
 public class GotestRunConfigurationProducer extends GoTestRunConfigurationProducerBase implements Cloneable {
   public GotestRunConfigurationProducer() {
     super(GotestFramework.INSTANCE);
   }
```

Comment:
This producer still should ignore GoMethodDeclarations but now it can do it overriding `findTestFunctionInContext` method